### ISSUE TITLE
Handling of negative test results is badly documented.

### ISSUE THREAD

**Issue by JensSteckhan** [corona-warn-app/cwa-documentation#417]
I took a test four days ago and added the test via QR code to the app (I am on iOS).
I was told by the DRK employees that I would be able to see the results within 48h both on the lab's website and in the Corona app.   
The website did display the negative result immediately. The app still thinks that "Test result is not yet available".
I called the lab repeatedly and got contradictory information ("we only maintain positive results", "you need to enter the result yourself", "we do not maintain results at all, but send them to the RKI, they do it"). I tried to find out how negative results in the app's documentation, but couldn't find anything.

Then I called the corona warn app hotline and got yet another information: some labs wouldn't maintain negative results, others would do it. And the lady at the hotline told me that this is intended.
I objected: why does the app tell me there are no results available then even if that's wrong, but that didn't impress the hotline.

## Where to find the issue
A proper documentation of negative results is completely missing

## Describe the issue
See above

**Comment by Ein-Tim** [comment]
Linked to #400

**Comment by daimpi** [comment]
Afaik negative test results should **always** be shown in CWA if everything is working correctly. But unfortunately not everything seems to be working correctly currently.
Here is the FAQ entry: https://www.coronawarn.app/en/faq/#qr_test
and here the GH issue: https://github.com/corona-warn-app/cwa-documentation/issues/400

One thing which doesn't happen in case of a negative result, is that you'd upload your keys to the server, b/c this is something which actually only happens in case you were infected. But that's a totally different thing, although I've seen some ppl confusing those issues before 😅.

**Comment by JensSteckhan** [comment]
Thanks for the link to the FAQ. If I am not mistaken, you cannot reach it directly from the App, right?
Is there anything you can do to end the misinformation? Both the lab and the Corona Warn App Hotline gave me wrong information. 
Also, if it is really true that a substantial amount of negative test results are not maintained, it might be a good idea to somehow manage the expectations in the app. Something along the lines ('around x % of test labs maintain the information properly. If your lab is one of them, you will see the test results here...')

**Comment by daimpi** [comment]
> Thanks for the link to the FAQ. If I am not mistaken, you cannot reach it directly from the App, right?

You're correct, that's not really possible: afaik you'd have to go to "App Information" -> Häufige Fragen which brings you to the FAQ page of the Bundesregierung… at the top there you find a link to the CWA FAQ. So it's quite complicated 😅. 

Regarding the misinformation situation: maybe someone from the CWA team could forward this to the appropriate places (e.g. the hotline)? @tkowark ? 🙂

**Comment by tkowark** [comment]
Yes, we are doing that and will continue to do until the situation improves. Thanks for sharing all the detailed concerns around this.

Quick addition to the FAQ: there's also a blog entry explaining some of the QR code scenarios: https://www.coronawarn.app/de/blog/2020-08-25-notes-qr-codes/

**Comment by MikeMcC399** [comment]
I believe the uncertainties are answered by the following:

- "I have scanned a QR code, but the test result could not be retrieved for days." https://www.coronawarn.app/en/faq/#qr_test
- "How does the app work?" https://www.coronawarn.app/en/ 
- "How do I enter a negative test result?" https://www.coronawarn.app/en/faq/#test_negative

If the lab is working with CWA QR codes it should report back positive and negative results to the app.

For remaining issues about this part of the process, there is a separate hot topic "Connectivity to laboratories / Transfer of QR code / QR code not working" https://github.com/corona-warn-app/cwa-documentation/issues/400 which remains open.

The app itself links to the FAQ https://www.bundesregierung.de/corona-warn-app-faq (German) or https://www.bundesregierung.de/corona-warn-app-faq-englisch (English).

There are some issues concerning the multiple FAQ sites (see
https://github.com/corona-warn-app/cwa-documentation/issues/289#issuecomment-752014364).

Perhaps this issue could be closed? If there are remaining suggestions for improving the FAQs, they could be logged under https://github.com/corona-warn-app/cwa-website/issues.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for your post.
@JensSteckhan Can we close this issue?

Best wishes, DS

  
--- 
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@dsarkar I suggest to close this issue due to missing author feedback and add the https://github.com/corona-warn-app/cwa-documentation/labels/stale label to it.
